libraries/node-core-library/src/LockFile.ts

```diff
@@ -257,14 +257,23 @@ export class LockFile {
 
     let lockFile: LockFile;
 
-    try {
-      // open in write mode since if this file exists, it cannot be from the current process
-      // TODO: This will malfunction if the same process tries to acquire two locks on the same file.
-      // We should ideally maintain a dictionary of normalized acquired filenames
-      lockFileHandle = FileWriter.open(pidLockFilePath);
-      lockFileHandle.write(startTime);
+    let currentBirthTimeMs: number;
 
-      const currentBirthTimeMs: number = FileSystem.getStatistics(pidLockFilePath).birthtime.getTime();
+    try {
+      try {
+        // open in write mode since if this file exists, it cannot be from the current process
+        // TODO: This will malfunction if the same process tries to acquire two locks on the same file.
+        // We should ideally maintain a dictionary of normalized acquired filenames
+        lockFileHandle = FileWriter.open(pidLockFilePath);
+        lockFileHandle.write(startTime);
+        currentBirthTimeMs = lockFileHandle.getStatistics().birthtime.getTime();
+      } catch (error) {
```

Comment:
Is there an error code that we can filter for? It feels a little uncomfortable swallowing all errors, including for `FileWriter.open`.

Reply:
I’ve reworked the PR to fix the root cause of the problem, there is no need in this try/catch block now, so I’ve removed it